applications/registration-react/src/pages/dataset-registration-page/registration-status/registration-status.component.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import {
+  isApproved,
+  isDraft,
+  isPublished
+} from '../../../lib/registration-status';
+import './registration-status.scss';
+import localization from '../../../services/localization';
+
+export const RegistrationStatus = ({ registrationStatus }) => (
+  <div className="fdk-registration-status">
+    {isDraft(registrationStatus) && (
+      <>
+        <img
+          src="/static/img/icon-draft-circle-md.svg"
+          className="ml-3 mr-2"
+          alt="icon"
+        />
+        <span>{localization.formStatus.draft}</span>
+      </>
```

Comment:
Den samme struktur gjentar seg for det to andre statuser. Er det mulig å redusere footprint av denne komponenten?

Reply:
Endret til en annen måte.